### PR TITLE
e2e: workaround: try-catch json parsing

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?
e2e: don't fail if the response is not a JSON object, just print it and keep going.

### CODE REVIEW

File: tests/e2e/utils/utils.js [hunk 1]
```diff
@@ -167,8 +167,15 @@ async function makeRequest(page, endpoint, apiVersion = "v0") {
...
     const resp = await fetch(url);
-    const jsonResp = await resp.json();
-    return jsonResp["data"];
+
+    try {
+      const jsonResp = await resp.json();
+      return jsonResp["data"];
+    }
+    catch(error) {
+      console.log("-- No JSON in response --");
```
Comment (pcrespov) [inline tests/e2e/utils/utils.js:176]: Can you log the text of the response? Ideally the headers and the body. This way we can see whether a) it really comes from "trafik" and b) the ``S`` comes from ``Service Unavailable``

**Review by pcrespov — COMMENTED** [review]
A rebase of this PR would also be good to remove all the commits above. Some tips on how to do this:

First make sure your master branch is cleanly in sync
```bash
# ensures current branch is master
git checkout master

# pulls all new commits made to upstream/master
git pull upstream master

# this will delete all your local changes to master
git reset --hard upstream/master

# take care, this will delete all your changes on your forked master
git push origin master --force
(You can define the original repo as "upstream" with git remote add upstream /url/to/original/repo.)
```

then 
```cmd
git checkout feature.print-status-II
git rebase master
```
should make the trick

File: tests/e2e/utils/utils.js (hunk 1 above)
Comment (odeimaiz) [reply tests/e2e/utils/utils.js:176]: The critical method that calls it is already logging the response, but I guess it doesn't harm if we also do it here.

**Review by sanderegg — APPROVED** [review]

**Review by GitHK — APPROVED** [review]
👍

**Pull request merged**